### DISCUSSION TITLE
Placeholder name of visitors

### DISCUSSION THREAD

**Discussion by vCren** [BG-Software-LLC/SuperiorSkyblock2#1303] [Pending Feature Requests] [answered]
I suggest adding a placeholder that will return a visitor list
**%superior_island_visitors_1%** name of first visitor, 2 - second etc

And 
**%superior_island_members_1%** same
It would return a list of island users

**Comment by Lemvs** [comment, marked as answer]
Implemented in latest dev build.

@OmerBenGera you can mark this answered and close it.
